cmd/agent/install_script.sh

```diff
@@ -64,10 +64,10 @@ if [ -n "$DD_HOST_TAGS" ]; then
     host_tags=$DD_HOST_TAGS
 fi
 
-if [ -n "$DD_URL" ]; then
-  dd_url=$DD_URL
+if [ -n "$REPO_URL" ]; then
+  repo_url=$REPO_URL
 else
-  dd_url="datadoghq.com"
+  repo_url="datadoghq.com"
```

Comment:
`dd_url` is still used here: https://github.com/DataDog/datadog-agent/blob/2dee057d1e3330dc39f45bc18ec4d06b07ee17b8/cmd/agent/install_script.sh#L227